example/tests.py

```diff
@@ -44,12 +47,16 @@ def assert_switch_inactive(browser, url=url):
                 'Switch is not inactive')
 
 
+def drop_datastore():
```

Comment:
Yeah, it was `Switch.c.drop()` before because Pymongo's Collection class supports a drop command. Datastore (`Switch.ds`) doesn't support a drop, so to make things easier I created a classmethod `Model.drop()`. If you look throughout the diff, you'll see all instances of `Switch.c.drop()` and `Model.c.drop()` changed to `Switch.drop()` and `Model.drop()`.